### ISSUE TITLE
JH - Standardizing Documentation

### ISSUE THREAD

**Issue by shashank-sharma** [justicehub-in/justicehub#1]
### README Documentation that needed:

- [ ] Explaining how JusticeHub works (Technical, possible to make wiki)
- [ ] Tags: chat|RocketChat, mailing|list, twitter in README at top (also remove unused ones)
- [ ] Contribution guidelines (Markdown file)
- [ ] Add SPOC in-case any urgent / vulnerability reporting

### Repository level changes:

- [ ] Issue template
- [ ] Adding metadata to JusticeHub repository (Links, info, tags)
- [ ] Standardized how to use labels in each issue (documentation, bug, enhancement etc)

### Extra work:

- [ ] Add good first issues for each repository (allow new comers to contribute) -> Will add in few repositories which we worked in
- [ ] If any questions, add it as issue, allow others to pitch in
- [ ] Need common RC Channel for others to come and interact (will link it with chat|RocketChat tags in documentation)

**Comment by apoorv74** [comment]
@shashank-sharma instead of using Rocket Chat for interactions, can we use the JH Forum? We can create a separate category for these converations such as `JH-Tech` or something similar and add content on how people can use that to discuss tech at JH, etc.

**Comment by shashank-sharma** [comment]
@apoorv74 That was my first impression, Initially I thought there was a possibility of RC but it's not possible, that's why I added discourse link in README.md (Can be visible as tag at top)

Now since Discourse is forum, anyone can create post over there right ? There are already categories defined so I don't think any changes is required as such

**Comment by apoorv74** [comment]
@shashank-sharma let's include the tag so its more visible. 

On Discourse:

- You need to signup for a JH account to get write access on the forum
- There isn't any category for tech-related conversations, so people might get confused on where to ask related queries. So, i'll suggest let's create a category and use that link for the Discourse tag on Github

**Comment by shashank-sharma** [comment]
@apoorv74 That works, I'll ask Abhinav to create one category (or if you can that would be great), then I'll update the link.
One thing what we should start doing is categorizing issues based on repository.

1. If justicehub level (like business logic, flow, process, decision) -> this repo works
2. If login is not working, auth etc -> then ckanext-emailauth
3. If some frontend error -> ckanext-justicehub_theme

I don't know how annoying it might become, but it's much better

**Comment by apoorv74** [comment]
@shashank-sharma i agree, we should do that. you can create this mapping as per your understanding and then we can review.

**Comment by shashank-sharma** [comment]
How I can see, issues can be:
Based on contributor:
1. Internal
2. External

Based on issue:
1. Business logic (example: login process should change)
2. Development
3. Vulnerability / major bug

Hence, 
internal  + Business logic = Taiga (should do research on making it open source if possible)
internal  + Development   = Github + Project board
external + Business logic = Forum
external + Development   = Github
(internal/external) + Vulnerability / Major bug = Mail / RC

Tools we will use:

1. Taiga - No issues there
2. Github issues - Will start maintaining and creating tags
3. Github Project board - Need research on this
4. Forum - Need to create category for development
5. Mailing - No issues, it should be active

Github categories:

1. Justicehub - Ideally business logic (if we move from taiga to github)

Main goal should be making this repository for whole CKAN + JH setup infra anywhere as development + production. It's a team call should we focus on this or not

2. ckanext-emailauth - Anything related to Authentication
3. ckanext-moderation - Anything for Dataset

Plugins will always be a separate entity, it should never be related to any specific project hence issues should be specific to that only. It will be maintainers responsibility to make sure that right issues are raised.
